Resources/public/js/views/dashboard/ez-base-dashboardblockview.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-base-dashboardblockview', function (Y) {
+    "use strict";
+    /**
+     * Provides the Dashboard Base Block View class
+     *
+     * @module ez-base-dashboardblockview
+     */
+    Y.namespace('eZ');
+
+    /**
+     * The dashboard base block view
+     *
+     * @namespace eZ
+     * @class DashboardBlockBaseView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.DashboardBlockBaseView = Y.Base.create('dashboardBlockBaseView', Y.eZ.TemplateBasedView, [], {
+        /**
+         * Renders the dashboard block view
+         *
+         * @method render
+         * @return {eZ.DashboardBlockView} the view itself
+         */
+        render: function () {
+            this.get('container').setHTML(this.template(this.getAttrs()));
```

Comment:
this view will not be rendered so no render here. Also, the rendering logic is completely up to each implementation.